blazor/rich-text-editor/import-export.md

```diff
@@ -174,9 +174,9 @@ Use the following code to export the RTF file. While cliking on the export butto
 
 ## Export to HTML file
 
-You can export the RTE content to the HTML format by using the [Syncfusion.DocIO](https://libraries.io/nuget/Syncfusion.DocIO.NET) libraries.
+Using the [Syncfusion.DocIO](https://libraries.io/nuget/Syncfusion.DocIO.NET) libraries, you can export RTE content to HTML format.
```

Comment:
Use [Syncfusion.DocIO](https://libraries.io/nuget/Syncfusion.DocIO.NET) libraries to export the RTE content to HTML format.